app/controllers/hq/partners_controller.rb

```diff
@@ -5,4 +5,11 @@ class Hq::PartnersController < ApplicationController
   def index
     @partners= Partner.all
   end
+
+  def show
+    if not (@partner= Partner.find_by_id(params[:id]))
```

Comment:
Re: `unless`
I avoided using it here because some members of the team dislike it.  Since I wasn't using it for a return value (the only time it's different than `if not`) I just omitted it